utils/cp_utils.py

```diff
@@ -60,14 +60,17 @@ def run_cellprofiler(
         analysis_run (bool, optional): will use functions to complete an analysis pipeline (default is False)
 
     """
-    print(f"Starting CellProfiler run on {pathlib.Path(path_to_images).name}")
-    # A log file is created for each plate or data set name (based on folder name with images)
-    with open(pathlib.Path(f"logs/cellprofiler_output_{pathlib.Path(path_to_images).name}.log"), "w") as cellprofiler_output_file:
-        # run CellProfiler for a illumination correction pipeline
-        command = f"cellprofiler -c -r -p {path_to_pipeline} -o {path_to_output} -i {path_to_images}"
-        subprocess.run(command, shell=True, stdout=cellprofiler_output_file, stderr=cellprofiler_output_file)
-    cellprofiler_output_file.close()
+    # run CellProfiler illumination correction pipeline
+    if not analysis_run:
```

Comment:
This might be semantics and my own misunderstanding but wanted to ask: should the comment or the variable name for `analysis_run` be changed in order to be more clear about what is occurring? Stated another way, are we not analyzing things when we run a CellProfiler pipeline in this if branch? Would something like `quality control` or `prepare data` be more accurate?

Reply:
I would think analysis run makes the most sense based on the format used by the Broad Institute for Cell Painting projects where they have a `analysis.cppipe` and a `illum.cppipe` file.

As well, if you are running an IC pipeline, you aren't actually analyzing anything per say, just preprocessing. While in an analysis pipeline, you do analyze and quantify morphology.

It is a semantics thing, but I think you make a great point bringing it up! Technically if you don't turn on analysis_run, the main point is that this CellProfiler run does not output a sqlite file but anything else so it doesn't just have to be an illum run it could be anything.